### PR TITLE
git-log: show branch graph

### PR DESCRIPTION
**Pull request by Larry850806**
----

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/git-log.md [hunk 1]
```diff
@@ -13,3 +13,7 @@
 - Show only the first line of each commits:
 
 `git log --oneline`
+
+- Show branch graph:
```
Comment (agnivade) [inline pages/common/git-log.md]: How about "Show the graph of commits in the current branch"  ?

Comment (Larry850806) [reply pages/common/git-log.md]: Thanks for your suggestion.

File: pages/common/git-log.md [hunk 2]
```diff
@@ -14,6 +14,10 @@
 
 `git log --oneline`
 
+- Show the graph of commits in the current branch:
```
Comment (sbrl) [inline pages/common/git-log.md]: Perhaps this will read better?

```suggestion
- Show a graph of commits in the current branch:
```

Comment (mfrw) [reply pages/common/git-log.md]: It seems I don't have permission to push to this repo :(

**Comment by agnivade** [comment]
There is a conflict due to merging master with this branch. After resolving that, we should be good to go.

**Comment by Larry850806** [comment]
Excuse me, I seldom send pull request. How do you know that there is a conflict? Github says that "This branch has no conflicts with the base branch."

**Comment by waldyrious** [comment]
Besides the merge conflict (which I also don't see on this side), I'd suggest placing this example immediately above the `git log --oneline --decorate --all --graph` one, as that produces a more logic sequence of examples.

**Comment by mfrw** [comment]
> Excuse me, I seldom send pull request. How do you know that there is a conflict? Github says that "This branch has no conflicts with the base branch."
> 
> 

Hi @Larry850806, when you had pushed it caused a merge conflict. I tried to correct it unsuccessfully.

File: pages/common/git-log.md (hunk 2 above)
Comment (sbrl) [reply pages/common/git-log.md]: @mfrw: I don't think the OP has given us permission to write the the branch of this PR.

File: pages/common/git-log.md [hunk 3]
```diff
@@ -18,6 +18,10 @@
 
 `git log --stat`
 
+- Show a graph of commits in the current branch:
+
+`git log --graph`
+
 - Show all commits, tags and branches for the entire repo in a graph format:
```
Comment (waldyrious) [inline pages/common/git-log.md]: For consistency with the newly introduced example:

```suggestion
- Show a graph of all commits, tags and branches in the entire repo:
```

**Comment by waldyrious** [comment]
Thanks for bearing with us during review, @Larry850806! LGTM, let's merge :)

**Pull request merged**